src/writeData/components/WriteDataHelperBuckets.tsx

```diff
@@ -68,17 +70,29 @@ const WriteDataHelperBuckets: FC<Props> = ({
     body = (
       <List
         backgroundColor={InfluxColors.Grey5}
-        style={{height: '200px'}}
+        style={
+          disabled
+            ? {
+                pointerEvents: 'none',
+                height: '200px',
+                cursor: 'not-allowed',
+                borderColor: '#1a1a2a',
+                backgroundColor: '#1a1a2a',
+                opacity: '0.5',
+                color: '#828294',
+              }
+            : {height: '200px'}
+        }
```

Comment:
I think we can move this into a variable or the stylesheet. I learned that it helps performance. 

something like : 
```
const listStyle = disabled
            ? {
                pointerEvents: 'none',
                height: '200px',
                cursor: 'not-allowed',
                borderColor: '#1a1a2a',
                backgroundColor: '#1a1a2a',
                opacity: '0.5',
                color: '#828294',
              }
            : {height: '200px'}
```      

and then using the variable in the component props.

Reply:
For reference : https://www.aaronbuxbaum.com/how-object-literals-hurt-performance/

Reply:
Yes I saw that, that was a good catch by Bucky!